src/main/java/fi/otavanopisto/kuntaapi/server/rest/ServicesApiImpl.java

```diff
@@ -16,7 +16,10 @@
 
 import fi.metatavu.kuntaapi.server.rest.ServicesApi;
 import fi.metatavu.kuntaapi.server.rest.model.Service;
+import fi.metatavu.kuntaapi.server.rest.model.ServiceLocationServiceChannel;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'fi.metatavu.kuntaapi.server.rest.model.ServiceLocationServiceChannel'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)